lib/spack/docs/gpg.rst

```diff
@@ -0,0 +1,43 @@
+.. _gpg:
+
+===========
+GPG support
+===========
```

Comment:
Wherever this ends up going, you'll want to add sections like:

```rst
.. _cmd-spack-gpg:

-------------
``spack gpg``
-------------

...

------------------
``spack gpg init``
------------------

...

------------------
``spack gpg sign``
------------------

...
```
The `.. _cmd-spack-gpg:` line is important so that it ends up adding a link to the [Command Index](http://spack.readthedocs.io/en/latest/command_index.html).